msteams-platform/toolkit/create-new-project.md

```diff
@@ -8,46 +8,71 @@ ms.topic: overview
+1. Select required **Capability**.
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/capability.png" alt-text="create1" border="true":::
+
+1. Select a programming language from the dropdown options.
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/program.png" alt-text="create2" border="true":::
+
+1. Enter an **Application name** and Press **Enter** (Application name must start with a letter and can only contain letters and digits.).
 
-### Create a new Teams app
+   :::image type="content" source="../assets/images/teams-toolkit-v2/app.png" alt-text="create3" border="true":::
 
-1. Open Visual Studio Code.
-1. Select the Teams Toolkit :::image type="icon" source="../assets/images/teams-toolkit-v2/teams-toolkit-sidebar-icon.PNG" border="true"::: icon in the Visual Studio Code sidebar.
-1. Select **Create a new Teams app**.
-1. Select from the available capabilities tab, bot, message extension, or a tab using SharePoint Framework (SPFx). 
-1. Select at least one option to start creating the Teams app.
 
-### Create a new Teams app using view samples
+### View samples
 
-You can create a new app by exploring **View samples** and selecting an existing sample. The selected sample may already have some functionality, for example a to-do list with an Azure backend, or an integration with the Microsoft Graph Toolkit.
+You can create a new Teams app by selecting an existing sample. Selecting an existing sample provides you with an inbuilt functionality. For example, a **Todo List with backend on Azure**, or **Hello World Bot with SSO**.<br>
 
- 1. Open **Teams Toolkit** from Microsoft Visual Studio Code.
- 1. Select **DEVELOPMENT** section in Treeview.
- 1. Select **View samples**. The sample gallery appears as shown in the following image:
+1. Select **Teams Toolkit** :::image type="icon" source="../assets/images/teams-toolkit-v2/teams-toolkit-sidebar-icon.PNG" border="true"::: icon in the Visual Studio Code sidebar.
+2. Select **DEVELOPMENT** section in Treeview and then Select **View samples**.
+   :::image type="content" source="../assets/images/teams-toolkit-v2/sample1.png" alt-text="View samples":::
+1. The **Samples** gallery displays.
 
-    :::image type="content" source="../assets/images/teams-toolkit-v2/view-samples.png" alt-text="View samples":::
+   :::image type="content" source="../assets/images/teams-toolkit-v2/samples2.png" alt-text="View samples2":::
+   
 
-You can explore and download samples and either run your app locally or remotely to preview in Teams web client. Follow the instructions for each sample, or select **View on GitHub** to open the sample within the `TeamsFx Samples repository` and browse the source code.
+Explore samples to quickly get started with concepts and code examples. Follow the instructions for each sample, or select **View on GitHub** to open the sample in the `TeamsFx Samples` repository and browse the source code.
```

Comment:
You can remove the world quickly. Please update information on the graphics and documents available to guide developers to create app.